### DISCUSSION TITLE
Removing errant value from PV1 segment

### DISCUSSION THREAD

**Discussion by richardlreynolds** [nextgenhealthcare/connect#6220] [Q&A] [answered]
I am trying to remove an errant string value from a PV1 segment. The string value 'null' is appearing in the PV1.20 field:

PV1||P|3437198^^WDO.ENDO/MOD^V|11|14100|||2819246^POLLOCK^J|2823044^ABDELAZIZ (Consultant)^H|320||||||||P|**null**|||||||||||||||||||||||||||N|||202406051545

I know that PV1.20 is a repeating field, and don't know if there is some method that I'm not aware of to ensure that the resultant segment ends up like:
PV1||P|3437198^^WDO.ENDO/MOD^V|11|14100|||2819246^POLLOCK^J|2823044^ABDELAZIZ (Consultant)^H|320||||||||P||||||||||||||||||||||||||||N|||202406051545

I've tried using javascript code like:
`delete msg['PV1']['PV1.20'];`
and 
`delete msg['PV1']['PV1.20']['PV1.20.1'];`

I've also tried `msg['PV1']['PV1.20']['PV1.20.1'] = '';`

**Comment by pacmano1** [comment, marked as answer]
that value is in pv1.19.1, actual message type irrelevant of course.  

![image](https://github.com/nextgenhealthcare/connect/assets/44065187/47c34fde-7b78-474a-af6f-089d0a9a217c)

**Reply by richardlreynolds** [reply]
Thanks for the prompt reply!

What is the code then that I need in the step (I have a step called PV1 to deal with changes needed to PV1) to ensure that this value is cleared/removed?

Would it be `delete msg['PV1']['PV1.19']['PV1.19.1'];` ?

**Reply by pacmano1** [reply]
I would normally just set it to an empty string.
